capture_hdr_loop.py

```diff
@@ -1,68 +1,65 @@
-"""
-Capture HDR frames in a loop (while actively changing some HDR settings).
+"""Capture HDR frames in a loop (while actively changing some HDR settings).
+
+Three sets of settings are chosen. These settings provide a dynamic range of
+about 9-10 stops, with a decent low- and highlight frame which is sufficient
+to grab good data on most scenes, including shiny for a medium camera operating
+at around 0.7m-1.2m
```

Comment:
space after number, e.g. 0.7 m to 1.2 m. ? 

P.S. Sorry, I was tortured while writing my thesis, can't help it...

Reply:
Hehe, I'll share blame with Theie (I copied the text from his comment in JIRA) :)